src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,28 @@
+.calendar {
+  padding: 10px;
+  width: #{$limit-width-of-row};
+  display: flex;
+  gap: $indent-of-days;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    text-decoration: none;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    align-items: center;
+    justify-content: center;
+    transition: 500ms;
+    display: none;
```

Comment:
You have set `display: none;` for the `&__day` element. This means that the days of the calendar will not be visible. If you want to hide specific days, you should apply this property to a specific class or use a modifier.